### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by nicosampler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,107 @@
...
+This repository contains a set of React components written in typescript.
+
+These components are being used to build [Safe-multisig](https://github.com/gnosis/safe-react) app.
+
+As Safe-multisig allows to integrate third party applications, Gnosis team recommends to build these apps using this components to get the following benefits:
+
+- Same style as the one used Safe-multisig. Making third-party apps looks native.
+- As the space used to render third-party apps is reduced, most of the components will be optimized to render properly in different resolutions.
+- Blockchain oriented: Some components will allow you to solve common problems like, inputs for ETH addresses and bigNumbers, Identicon images, etc.
+- Safe time, as you don't have to start from scratch building your safe-app.
```
Comment (fernandomg) [inline README.md:17]: typo
```suggestion
- Save time, as you don't have to start from scratch building your safe-app.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,107 @@
...
+
+## How to install
+
+```bash
+   yarn add @gnosis.pm/safe-react-components
+```
+
+## Integration
+
+This library make use of [material-ui](https://material-ui.com/) as a peerDependency, it means you must install it in your Safe-app. Make sure to provide the same version as the one being use by the current version of this library.
```
Comment (fernandomg) [inline README.md:28]: typos
```suggestion
This library makes use of [material-ui](https://material-ui.com/) as a `peerDependency`, it means you must install it in your Safe-app. Make sure to provide the same version as the one being used by the current version of this library.
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,107 @@
...
+
+Once everything is installed, you have to instantiate a [ThemeProvider](https://styled-components.com/docs/api#themeprovider) from [styled-components](https://@gnosis.pm/safe-react-components/).
+
+This example uses the theme exported by safe-react-components. Here, you could extends this theme to customize it to your needs.
```
Comment (fernandomg) [inline README.md:32]:
```suggestion
This example uses the theme exported by safe-react-components. Here, you can extend this theme to customize it to your needs.
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,107 @@
...
+export default () => (
+  <ThemeProvider theme={theme}>
+    <App />
+  </ThemeProvider>
+);
+```
+
+### Using the same fonts as Safe-multisig
+
+If you want your app has the same fonts as the one used by Safe-multisig you need to do next.
```
Comment (fernandomg) [inline README.md:49]:
```suggestion
If you want your app to have the same fonts as the one used by Safe-multisig you need to do the following.
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,107 @@
...
+        src: local('Averta'), local('Averta Bold'),
+        url(${avertaFont}) format('woff2'),
+        url(${avertaBoldFont}) format('woff');
+    }
+`;
+
+export default GlobalStyle;
+```
+
+And then include it wrapping your application.
```
Comment (fernandomg) [inline README.md:68]: Not sure about the _wrapping_ as it's not wrapping it, but at the same `App` level.

**Comment by rmeissner** [comment]
Can we add what version of material-ui is required ... looking in the package.json is annoying :P

**Comment by rmeissner** [comment]
styled-components is also a peer dependency, right?

File: README.md [hunk 6]
```diff
@@ -1,3 +1,107 @@
...
+
+```bash
+   yarn add @gnosis.pm/safe-react-components
+```
+
+## Integration
+
+This library makes use of [material-ui](https://material-ui.com/) as a `peerDependency`, this means you must install it in your Safe App. Make sure to provide the same version as the one being used by the current version of this library.
+
+Once everything is installed, you have to instantiate a [ThemeProvider](https://styled-components.com/docs/api#themeprovider) from [styled-components](https://@gnosis.pm/safe-react-components/).
```
Comment (rmeissner) [inline README.md:30]: why does `styled-components` link to `https://@gnosis.pm/safe-react-components/`

File: README.md [hunk 7]
```diff
@@ -1,3 +1,107 @@
...
+- **Native feel:** Build your Safe Apps with a similar style as the one used Safe-multisig. This makes your Safe Apps feel almost like a native feature of the Safe Multisig.
+- **Responsive:** Most of the components will are optimized to render properly in different resolutions.
+- **Blockchain-focused:** Some components solve common blockchain-related problems like inputs for ETH addresses and bigNumbers, identicon images, and more.
+- **Save time:** No need to build all components from scratch.
+
+
+## How to install
+
+```bash
+   yarn add @gnosis.pm/safe-react-components
```
Comment (mikheevm) [inline README.md:23]: we should also include npm command

File: README.md [hunk 8]
```diff
@@ -1,3 +1,107 @@
...
+import { createGlobalStyle } from 'styled-components';
+import avertaFont from '@gnosis.pm/safe-react-components/dist/fonts/averta-normal.woff2';
+import avertaBoldFont from '@gnosis.pm/safe-react-components/dist/fonts/averta-bold.woff2';
+
+const GlobalStyle = createGlobalStyle`
+    @font-face {
+        font-family: 'Averta';
+        src: local('Averta'), local('Averta Bold'),
+        url(${avertaFont}) format('woff2'),
+        url(${avertaBoldFont}) format('woff');
```
Comment (mikheevm) [inline README.md:61]: Please include font-display swap in this example

https://developers.google.com/web/updates/2016/02/font-display

**Comment by nicosampler** [comment]
Changes pushed! thx for the review!

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr18--safereactcomponents.review.gnosisdev.com

**Pull request merged**